lms/djangoapps/discussion_api/views.py

```diff
@@ -90,6 +90,8 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
 
         * page_size: The number of items per page (default is 10, max is 100)
 
+        * topic_id: The topic, if given, to retrieve the threads for
```

Comment:
This should mention that the parameter can be used multiple times to retrieve threads from multiple topics at once